.github/workflows/reusable-eslint.yml

```diff
@@ -0,0 +1,62 @@
+###########################################################
+# Copyright (C) 2022 IOTech Ltd
+###########################################################
+
+name: eslint
+
+on:
+  workflow_call:
+    inputs:
+      WORKING_DIRECTORY:
+        description: The working directory, default is project root.
+        required: false
+        type: string
+        default: "."
+      USE_PRIVATE_MODULE:
+        description: If true, add the github private access token to .npmrc file for the private node modules.
+        required: false
+        type: boolean
+        default: false  
+      REPORTER:
+        description: Reporter of reviewdog command [github-pr-check,github-check,github-pr-review]. Default is github-pr-review.
+        required: false
+        type: string
+        default: "github-pr-review"
+      ESLINT_FLAGS:
+        description: Flags and args of eslint command. Default is "."
+        required: false
+        type: string
+        default: "."
+
+permissions:
+  contents: read
+ 
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+    env:
+      NODE_VERSION: 16
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+      # Install Node
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      # Set up environment for private modules
+      - name: Set up for private modules
+        if: ${{ inputs.USE_PRIVATE_MODULE }}
+        run: |      
+          echo '//npm.pkg.github.com/:_authToken=${{ secrets.NPM_PRIVATE_PAT }}' >> .npmrc
+        working-directory: ${{ inputs.WORKING_DIRECTORY }}
+      # Install Node packages and dependencies
+      - name: Install dependencies
+        run: npm ci
+        working-directory: ${{ inputs.WORKING_DIRECTORY }}
+      # Run ESLint checks
+      - uses: reviewdog/action-eslint@v1
+        with:
```

Comment:
Could you please add`github_token: ${{ secrets.JENKINS_PAT }}` to see if it can fix the permission issue in edgexpert-manager?